bench/.env.example

```
@@ -0,0 +1,27 @@
+# Run `nproc` to get CPU count.
+#
+# Reserve first two CPUs (0-1) for k6 + aux services
+AUX_CPUSET=0-1
+
+# Filecoin Node runs on the remaining CPUs
+# Example for 8 logical CPUs: 2-7
+NODE_CPUSET=2-7
+
+# Snapshot (shared)
+LATEST_SNAPSHOT_URL=https://forest-archive.chainsafe.dev/latest/mainnet/
+ARIA2_CONN=5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ARIA2_CONN key should go before the LATEST_SNAPSHOT_URL key

(UnorderedKey)